qemu/tests/cfg/blockdev_commit_non_existed_base.cfg

```diff
@@ -0,0 +1,36 @@
+- blockdev_commit_non_existed_base:
+    type = blockdev_commit_non_existed_base
+    virt_test_type = qemu
+    only Linux
+    images += " data"
+    force_create_image_data = yes
+    force_remove_image_data = yes
```

Comment:
use remove_image_data = yes instead, force_remove_image_data doesn't work for removing image, in your case data was removed finally just because vt sets remove_image = yes when removing image

Reply:
Thanks for your guidance,updated the code.